scss/_patterns_links.scss

```diff
@@ -70,6 +70,7 @@ $external-icon-size--headings: #{map-get($icon-sizes, heading-icon--x-small)};
       background-color: currentColor;
       content: '\00A0';
       display: inline-block;
+      line-height: 1em;
       // stylelint-disable property-no-vendor-prefix
       -webkit-mask: url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' width='24' height='24' %3E%3Cg fill='none' fill-rule='evenodd'%3E%3Cpath d='M-1-1h16v16H-1'/%3E%3Cpath fill='%23111' d='M4.867 1.313C.6 1.32.067 1.443.067 4.51v6.4c0 3.2.533 3.2 5.333 3.2h2.133c4.8 0 5.334 0 5.334-3.2v-1.6h-1.6v1.068c0 2.133 0 2.133-4.267 2.133H5.933c-4.266 0-4.266 0-4.266-2.132V5.044c0-1.93.034-2.112 3.2-2.13v-1.6z'/%3E%3Cpath d='M-1-1h16v16H-1'/%3E%3Cpath fill='%23111' d='M6.435 2.16c.11-.446 7.113-2.196 7.448-1.86.335.334-1.416 7.335-1.863 7.447-.447.112-5.697-5.14-5.586-5.586z'/%3E%3Cpath fill='%23111' d='M9.032 3.38L4.705 7.708l1.767 1.767L10.8 5.148'/%3E%3C/g%3E%3C/svg%3E")
         no-repeat 0.25em 0 / #{$mask-size};
```

Comment:
Not entirely related, but as we update the link we can verify that:

Size of the mask is `$mask-size` here, for `-webkit-mask`. Couple lines below, in unprefixed `mask` it uses `$size` instead. This makes the external icon use different size in Firefox and webkit based browsers.

Is this expected?

Reply:
Firefox (1rem):

<img width="837" alt="Screenshot 2020-05-20 at 07 50 57" src="https://user-images.githubusercontent.com/83575/82409781-b9a68700-9a6e-11ea-9a3f-a5d81eb0651b.png">

Chrome (0.6667rem):

<img width="833" alt="Screenshot 2020-05-20 at 07 51 02" src="https://user-images.githubusercontent.com/83575/82409791-bf03d180-9a6e-11ea-9ebd-1c708515d6dd.png">